docs/core/Graph.md

```diff
@@ -39,6 +39,7 @@ Graph is a set of graphs that share a common set of base visualization structure
 | dimension              | object   | {}                                             | Refer [Dimension](#dimension)                                                                                                                                                               |
 | legendPadding          | object   | `{ top: 4, bottom: 4, left: 8, right: 8 }`     | Sets the legend padding and removes the carbon default margin of 8px around legend                                                                                                          |
 | locale                 | object   | LOCALE.en_US                                   | Locale object for X-Axis tick values                                                                                                                                                        |
+| opaqueBackground       | false    | false                                          | Toggle between a transparent or White background for the container                                                                                                                          |
```

Comment:
Issue [#35](https://github.com/cerner/carbon-graphs/issues/35) description says `Implement default white background for carbon graphs`

Should we make this prop to `true` by default so that it will default white background?

Reply:
We decided to make transparent the default to make this change passive in the TDD.

Reply:
Probably we can update the [issue description]([#35](https://github.com/cerner/carbon-graphs/issues/35))

Reply:
Agreed that it would be good to update the issue but I would prefer to keep the original description but add this as a note that "after consideration, the default background will continue to be transparent in order to be passive to existing consumers but a white option will be provided."